libs/libnx/Makefile

```diff
@@ -22,6 +22,8 @@
 
 include $(TOPDIR)/Make.defs
 
+ifeq ($(CONFIG_NX),y)
```

Comment:
`CONFIG_NXFONTS` can be used without `CONFIG_NX`

Reply:
Done